data/tasks.txt

```diff
@@ -0,0 +1,4 @@
+T | 1 | CS2107 Revision | 
```

Comment:
Hi Wei Xue! You can try to delete the output lines before committing, just in case it affects with testing.